lldb/include/lldb/Target/Statistics.h

```diff
@@ -130,10 +130,14 @@ struct ConstStringStats {
   ConstString::MemoryStats stats = ConstString::GetMemoryStats();
 };
 
+struct StatisticsOptions {
+  bool summary_only = false;
+};
+
 /// A class that represents statistics for a since lldb_private::Target.
 class TargetStats {
 public:
-  llvm::json::Value ToJSON(Target &target);
+  llvm::json::Value ToJSON(Target &target, bool summary_only = false);
```

Comment:
We should still pass in a "const StatisticsOptions options = StatisticsOptions()" instead of a boolean here. If we add more options in the future, we should avoid changing this API that way.